### ISSUE TITLE
ansible-galaxy init pulling in "ignored files"

### ISSUE THREAD

**Issue by daniel-wtd** [ansible/galaxy#315]
Today I wanted to play around with custom skeletons as documented here: 
https://docs.ansible.com/ansible/latest/galaxy.html#using-a-custom-role-skeleton

## Actual Behaviour:

When running `ansible-galaxy init --role-skeleton=/path/to/skeleton role_name`:
- The directories and files from the custom skeleton are pulled in.
- variable "role_skeleton_ignore" seems to be ignored

## Expected Behaviour:

When running `ansible-galaxy init --role-skeleton=/path/to/skeleton role_name`:
- The directories and files from the custom skeleton are not pulled in as described in the docs (.git, .gitkeep).
- variable "role_skeleton_ignore" works as designed

## Steps to reproduce:

1. create a *skeleton* directory
2. cd in the *skeleton* directory
2. `git init`
3. `touch .gitkeep`
4. `ansible-galaxy init --role-skeleton=/path/to/skeleton role_name`
5.  `ls -la role_name`
6. .git still existing

## Information:

OS: Fedora 27
```
ansible --version
ansible 2.4.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ds/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.14 (default, Jan 17 2018, 14:28:32) [GCC 7.2.1 20170915 (Red Hat 7.2.1-2)
```

**Comment by chouseknecht** [comment]
Think I found the issue. Submitted a PR here: https://github.com/ansible/ansible/pull/35744

**Comment by chouseknecht** [comment]
PR merged to Ansible 2.5

**Comment by daniel-wtd** [comment]
awesome, thanks a lot.
